src/uploader/uploader.service.test.ts

```diff
@@ -0,0 +1,24 @@
+import FilesUploaderService, {
+  IFilesUploaderService,
+} from "./uploader.service";
+
+describe("FilesUploaderService", () => {
+  let filesUploaderService: IFilesUploaderService;
+  beforeEach(() => {
+    filesUploaderService = new FilesUploaderService();
+  });
+
+  describe("uploadFile", () => {
+    it("should return message when successfully uploaded file", async () => {
+      const filePath = "C:/Users/michal/Desktop/invoices/invoice_1.pdf";
+      const uploadTo =
+        "https://drive.google.com/drive/u/0/folders/1UiU4lsu5-9dr7hAqwUmk4CEL9sjTyAgU";
+      expect(
+        await filesUploaderService.uploadFile({
+          filePath,
+          uploadTo,
+        })
+      ).toBe(`invoice_1.pdf has been successfully uploaded to ${uploadTo}`);
+    });
+  });
+});
```

Comment:
Co za dużo testów to niezdrowo 🍰 

Zmarnowałeś czas na test który sprawdza czy funkcja zwraca poprawnego stringa....
Gdyby była tutaj jakaś logika biznesowa - jak najbardziej pisanie testu ma sens, lecz w tym przypadku chyba masz troche za dużo czasu.

Reply:
nie testuj implementacji, testuj logikę biznesową. 1. albo nie pisać testów, 2. w przyszłości pewno będzie tu biblioteka, stworzyć abstrakcję i sprawdzić, czy został wywołany.
3. sprawdzić, czy dostał dobre argumenty

Reply:
zrobiony update